src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -95,6 +95,13 @@ def validate(self, attrs):
         # project_rule(_details) endpoints by setting it on attrs
         actions = attrs.get("actions", tuple())
         for action in actions:
+            # XXX(colleen): For rules that have the Jira integration as an action
+            # we need to ensure the user has selected a Jira project and issue type
+            if action["id"] == "sentry.integrations.jira.notify_action.JiraCreateTicketAction":
```

Comment:
@ceorourke we don't have the same problem with Azure? Either way, this feels a bit too hacky for me.